jupyterhub/handlers/base.py

```diff
@@ -479,7 +490,8 @@ def finish_user_spawn():
             if status is not None:
                 toc = IOLoop.current().time()
                 self.statsd.timing('spawner.failure', (toc - tic) * 1000)
-                raise web.HTTPError(500, "Spawner failed to start [status=%s]" % status)
+                raise web.HTTPError(500, "Spawner failed to start [status=%s]. The logs for %s may contain details." % (
```

Comment:
@minrk a general question re: logs.

 if I am using multiple servers per user, will I be able to tell which one failed to be spawned?

Reply:
I believe so, yes. When we `raise HTTPError`, this is logged as `500 POST /users/:name/server`. If one HTTP endpoint can prompt the creation of different servers, then this might not be enough.